diff --git a/node_modules/toastify-react-native/components/ToastManager.tsx b/node_modules/toastify-react-native/components/ToastManager.tsx
index bb82244..1a1500c 100644
--- a/node_modules/toastify-react-native/components/ToastManager.tsx
+++ b/node_modules/toastify-react-native/components/ToastManager.tsx
@@ -65,6 +65,10 @@ class ToastManager extends Component<ToastManagerProps, ToastManagerState> {
     ToastManager.__singletonRef?.show(text, Colors.error, "alert-circle", position);
   };
 
+  static hideAll = () => {
+    ToastManager.__singletonRef?.hideToast()
+  };
+
   show = (text = "", barColor = Colors.default, icon: string, position?: ToastManagerProps["position"]) => {
     const { duration } = this.props;
     this.state.barWidth.setValue(this.props.width);
@@ -188,20 +192,10 @@ class ToastManager extends Component<ToastManagerProps, ToastManagerState> {
             },
           ]}
         >
-          {showCloseIcon && (
-            <TouchableOpacity onPress={this.hideToast} activeOpacity={0.9} style={styles.hideButton}>
-              <Icon name="close-outline" size={22} color={Colors[theme].text} />
-            </TouchableOpacity>
-          )}
           <View style={styles.content}>
             <Icon name={icon} size={22} color={barColor} style={styles.iconWrapper} />
             <Text style={[styles.textStyle, { color: Colors[theme].text, ...textStyle }]}>{text}</Text>
           </View>
-          {showProgressBar && (
-            <View style={styles.progressBarContainer}>
-              <Animated.View style={{ width: barWidth, backgroundColor: barColor }} />
-            </View>
-          )}
         </View>
       </Modal>
     );
diff --git a/node_modules/toastify-react-native/index.ts b/node_modules/toastify-react-native/index.ts
index 969b048..0208422 100644
--- a/node_modules/toastify-react-native/index.ts
+++ b/node_modules/toastify-react-native/index.ts
@@ -5,6 +5,7 @@ export const Toast = {
   success: ToastManager.success,
   warn: ToastManager.warn,
   error: ToastManager.error,
+  hideAll: ToastManager.hideAll
 };
 
 export default ToastManager;
